frontend/.yarnrc

```diff
@@ -0,0 +1 @@
+yarn-path ".yarn/releases/yarn-1.15.2.js"
```

Comment:
I'd expect yarn.lock changes to add integrity checks, but maybe those slipped into master by mistake.

Reply:
It seems that existing `yarn.lock` entries are not automatically updated to the latest format upon `yarn install`, regardless of using `--update-checksums` parameter (tested with Yarn 1.15.2).

However, when you add or modify a dependency, that dependency (as well as any related ones) will have their corresponding `yarn.lock` entries updated to the latest format.

Only a few entries in the current `yarn.lock` already have an `integrity` field. This indicates that those dependencies were added or modified using a newer version of Yarn and passed code review (people oftentimes skip over `yarn.lock` changes).